Jenkinsfile_CNP

```diff
@@ -116,7 +116,7 @@ withPipeline(type, product, component) {
             environmentOfDependencies = "aat"
         }
         env.DEFINITION_STORE_HOST      = "http://ccd-definition-store-api-${environmentOfDependencies}.service.core-compute-${environmentOfDependencies}.internal"
-        env.ELASTIC_SEARCH_FTA_ENABLED = "true"
```

Comment:
Elastic search befta has been enable for prs,  in dev branch.  The correct values has to be fixed in another pr, or this one . ☝️